themes/mms-cloud/page.html

```diff
@@ -59,6 +59,21 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
     </div>
   {%- endif %}
 
+  {%- if theme_backup_monitoring_eol %}
+    <div class="alert alert-info alert-dismissible fade in" role="alert" style="margin-left: 20px">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">Cloud 
+          Manager will no longer support Backup and Monitoring for 
+          MongoDB 2.6, 3.0, 3.2, or 3.4 after September 30, 2022.
+          Please <a href="{{theme_base_url}}/cloud-manager/tutorial/change-mongodb-version/">upgrade your MongoDB deployment</a>
+          or <a href="{{theme_base_url}}/atlas/migration-to-atlas-choosing/">migrate to Atlas.</a>
+        </h4>
```

Comment:
**issue:** Simplify the language: shorten the headline, be direct in the change, be explicit in the call to action without pleading.
```suggestion
        <h4 class="alert-heading" style="margin-bottom: 10px;">Backup, Monitoring Support on MongoDB 3.4 and Earlier Ends September 30, 2022</h4>
         Cloud Manager ends support Backup and Monitoring for MongoDB 2.6, 3.0, 3.2, or 3.4 on Friday, September 30, 2022.
         <a href="{{theme_base_url}}/cloud-manager/tutorial/change-mongodb-version/">Upgrade your MongoDB databases to 3.6 or later</a>
         or <a href="{{theme_base_url}}/atlas/migration-to-atlas-choosing/">migrate your databases to MongoDB Atlas.</a>

```